packages/features/ee/teams/components/CreateTeamEventType.tsx

```diff
@@ -0,0 +1,41 @@
+import type { EventType } from "@prisma/client";
+import { useRouter } from "next/navigation";
+
+import { TeamEventTypeForm } from "@calcom/features/ee/teams/components/TeamEventTypeForm";
+import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Button } from "@calcom/ui";
+
+export const CreateTeamEventType = () => {
+  const router = useRouter();
+  const searchParams = useCompatSearchParams();
+  const { t } = useLocale();
+
+  const teamId = searchParams?.get("id") ? Number(searchParams.get("id")) : -1;
+
+  const handleSuccessMutation = (eventType: EventType) => {
+    router.push(`/settings/teams/${teamId}/profile`);
+  };
```

Comment:
`CreateTeamEventType` is doing very specific thing related to onboarding.
e.g. reading teamId from id param, navigating to Profile page. I think we should keep this component inside onboarding specific file.